internal/engine/local/controller.go

```diff
@@ -3,28 +3,35 @@ package local
 import (
 	"context"
 	"fmt"
+	"io"
 	"sync"
 	"time"
 
+	"github.com/tilt-dev/probe/pkg/probe"
+	"github.com/tilt-dev/probe/pkg/prober"
+	v1 "k8s.io/api/core/v1"
+
 	"github.com/tilt-dev/tilt/internal/store"
 	"github.com/tilt-dev/tilt/pkg/logger"
 	"github.com/tilt-dev/tilt/pkg/model"
 	"github.com/tilt-dev/tilt/pkg/model/logstore"
 )
 
 type Controller struct {
-	execer    Execer
-	procs     map[model.ManifestName]*currentProcess
-	procCount int
+	execer       Execer
+	procs        map[model.ManifestName]*currentProcess
+	procCount    int
+	probeManager ProberManager
```

Comment:
Oops, yeah, this should be `proberManager`.

I agree the naming is confusing and sorta-inherited from the original K8s code; I tried to tighten it up to be precise without deviating too much -- prober is the code that performs a status check, probe is the spec.

Reply:
Cleaned up in 05031d62383c2fd6ebcf8f7923fcf053bf58f5ac - let me know if you see any other errant misnomers or just generally have suggestions on the naming to make it clear without diverging massively from K8s (which I think would cause _more_ confusion)